tests/client.test.ts

```diff
@@ -68,12 +73,49 @@ describe('CloudFunctionsClient', () => {
       const cf: CloudFunction = {
         name: testFunctionName,
         runtime: 'nodejs16',
-        environmentVariables: { KEY1: 'VALUE1' },
-        buildEnvironmentVariables: { KEY1: 'VALUE1' },
-        entryPoint: 'helloWorld',
+        description: 'test function',
         availableMemoryMb: 512,
+        buildEnvironmentVariables: { BUILDKEY1: 'VALUE1', BUILDKEY2: 'VALUE2' },
+        // buildWorkerPool: string,
```

Comment:
These are the "not feasible in this PR" fields. I didn't want to boil the ocean.